tsa-test/src/test/resources/checkers/on-out-message-test/checker.fc

```diff
@@ -0,0 +1,29 @@
+#include "../../imports/stdlib.fc";
+#include "../../imports/tsa_functions.fc";
+
+global int visited;
+
+() on_out_message(cell msg_full, slice msg_body, int contract_id) impure method_id {
```

Comment:
Since this handler is called after action phase, we should probably add receiver's `contract_id` here as well. If the message isn't received by other analyzed contract, we can make some special value for that (for example, -1).